### DISCUSSION TITLE
Is there any way to use classes ('white') instead of setColor('#fff')?

### DISCUSSION THREAD

**Discussion by SoftEng-Islam** [ueberdosis/tiptap#4407] [General Questions] [answered]
This code inject **inline css style**.
`editor.value?.chain().focus().setColor('#fff').run();`

And i want my Text Element take a **_class_**
Like this `<span class="text-red-500">my text</span>`

_I have a lot of colors so i need flexible solution_

My project use => Vue3 typescript tailwindcss tiptap

![image](https://github.com/ueberdosis/tiptap/assets/142545372/b1165f1c-5be9-4e3a-be5c-8380edb4b90e)

**Comment by trebitowski** [comment, marked as answer]
Make your own color extension (or extend the existing one)

https://github.com/ueberdosis/tiptap/blob/develop/packages/extension-color/src/color.ts

**Reply by SoftEng-Islam** [reply]
Thank you trebitowski
